src/testController.ts

```diff
@@ -150,6 +157,23 @@ export class TestController {
     });
   }
 
+  private async rerunLastTest() {
+    if (!this.lastTest) {
+      return;
+    }
+
+    // TODO: Add telemetry for this?
```

Comment:
If so, do we want to send a "rerun_last_event" in addition to the existing `"test"` or `"test_in_terminal"` command?